### DISCUSSION TITLE
title 动态表格 设置Height="500" IsFixedHeader="true"后 查询结果不显示

### DISCUSSION THREAD

**Discussion by baij45** [dotnetcore/BootstrapBlazor#4147] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

`<Table TItem="DynamicObject" DynamicContext="DataTableDynamicContext"
       IsBordered="true" IsMultipleSelect="false" ShowToolbar="true" ShowRefresh="false" ShowDefaultButtons="false" IsPagination="true" ShowExportButton="false"
       OnExportAsync="OnExportAsync" OnSort="OnSort" ShowFooter="true" ShowLoading="true" IsStriped="true"
       PageItemsSource="new int[]{1000,2000,3000,4000,5000,10000}" TableSize="TableSize.Compact" FixedMultipleColumn="true"
      >`
![544$J7F0F6${YD3@YU7 ZH1](https://github.com/user-attachments/assets/65249522-7cb8-47af-8392-988ae19aaa7f)

> 动态表格不设置 Height="500" IsFixedHeader="true" 能正常显示数据

`<Table TItem="DynamicObject" DynamicContext="DataTableDynamicContext"
       IsBordered="true" IsMultipleSelect="false" ShowToolbar="true" ShowRefresh="false" ShowDefaultButtons="false" IsPagination="true" ShowExportButton="false"
       OnExportAsync="OnExportAsync" OnSort="OnSort" ShowFooter="true" ShowLoading="true" IsStriped="true"
       PageItemsSource="new int[]{1000,2000,3000,4000,5000,10000}" TableSize="TableSize.Compact" FixedMultipleColumn="true"
       Height="500" IsFixedHeader="true">`
![BV_T5DK9BMQ%`R7( ($YGK](https://github.com/user-attachments/assets/ab7d67ae-07e8-4afb-9acf-47d72c0e07fd)

> 加上 Height="500" IsFixedHeader="true"之后，表格就不显示数据

**Comment by bb-auto** [comment]
@baij45 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply]
@baij45 未复现
